tests/test_expt_file_counts_handler.py

```diff
@@ -23,7 +23,8 @@ def test_put_expt_file_counts_request_dict():
             'file_type_name': 'example_type',
             'file_extension': '.example',
             'time_valid': '2023-02-05 06:00:00',
-            'forecast_length' : 120,
+            'forecast_hour' : 120,
+            'file_size_bytes': 1203949459, 
```

Comment:
can we make test input to be something that is bigger than a single precision float? Sorry i got lost in significant digits here. maybe it already is

Reply:
yes, I'll update it, it is short a digit.